applications/portal/backend/api/import_export/AntibodyResource.py

```diff
@@ -160,11 +163,18 @@ def save_instance(self, instance, is_create, using_transactions=True, dry_run=Fa
                 # we don't have transactions and we want to do a dry_run
                 pass
             else:
-                instance.save(update_search=False)
+                self.instances.append(instance)
+                # instance.save(update_search=False, from_import=True)
+                # instance.save.__wrapped__(instance, update_search=False, from_import=True)
         self.after_save_instance(instance, using_transactions, dry_run)
 
     def after_import(self, dataset, result, using_transactions, dry_run, **kwargs):
         super().after_import(dataset, result, using_transactions, dry_run, **kwargs)
+        from multiprocessing.pool import ThreadPool
+
+        with ThreadPool() as pool:  # Set by default to the number of CPUs
```

Comment:
The real computation is in the db, not here. As the number of cpus is 1 in the deployment I'd set a fair higher number, say 8